### PR TITLE
Optimization Site Audit report

### PR DESCRIPTION
**Pull request by davidnguyen88**
GH: https://github.com/ConsenSys/dx-team/issues/970

We optimized the Site Audit report from Semrush SEO Audit.

You can preview the build output on the link: https://bafybeiflzjzgk2q5qancrytce6fnhsc6gvrw7wyr5cowjwi7qqjjkxgm6u.on.fleek.co/

### CODE REVIEW

File: src/events/index.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ---
 template: events.html
-title: Truffle Suite
+title: Truffle Events
+description: Truffle Events
```
Comment (davidmurdoch) [inline src/events/index.md:4]:
```suggestion
```

File: src/policy/index.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 ---
 template: policy.html
-title: Truffle Suite
+title: Truffle Policy
+description: This privacy policy (“Policy”) describes how ConsenSys Software Inc.
```
Comment (davidmurdoch) [inline src/policy/index.md:4]:
```suggestion
```

**Review by davidmurdoch — CHANGES_REQUESTED** [review]

**Comment by jlazoff** [comment]
Hi @davidmurdoch , 

Thank you for your code review above. @davidnguyen88 will be looking into them, and applying them globally. Thank you @davidnguyen88 and @cuonglt-ync for your work on this ticket. 

@davidmurdoch @kevinbluer : Sunny, Dana and I were debugging an issue flagged by GSC. None of the pages on the site were being indexed. Upon further inspection, I found that there is a deep tangle to issues on how the pages are being redirected, and used. I have written a couple of examples, and then will propose a plan to fix this issue. Our team can help you standardize this all. 

1. There is a 301 Permanent Redirect from trufflesuite.com/docs to trufflesuite.com/docs/ - this is permanent, and good, since we will want to have all pages standardized with a "/" ending, except the homepage. 
2. The canonical URLs of all the pages have to be modified to the new "/" ending. At the moment the canonical URL includes a "/index.html" ending. Here is a map of the URL endings I was able to find, and their behavior:

- Canonical is always: <link rel="canonical" href="https://trufflesuite.com/guides/index.html">
- https://trufflesuite.com/guides/index.html - link used on menu
- https://trufflesuite.com/guides/ - works
- https://trufflesuite.com/guides redirects to https://trufflesuite.com/guides/
- https://trufflesuite.com/guides.html - does not work in this case, but does in others 
- The sitemap includes the https://trufflesuite.com/guides/index.html invalidating all the URLs on Google Search Console

Here is a way to standardize and save each pages SEO power:

Our goal is to have every page, expect the homepage, with a "/" ending, such as: https://trufflesuite.com/guides/

**Comment by jlazoff** [comment]
Hi @kevinbluer,
Can you share securely with @cuonglt-ync the following:

Screen Shot 2022-03-21 at 17 50 27

Appreciate your help!,
Joshua

**Comment by davidmurdoch** [comment]
@jlazoff I'm working accessibility changes in a new branch. This includes changes to the default URLs (canonicalizing the directory "/" as opposed to `index.html`). I'll take care of all of the changes you've requested above and ping you when it's ready. Thanks!

**Comment by jlazoff** [comment]
> @jlazoff I'm working accessibility changes in a new branch. This includes changes to the default URLs (canonicalizing the directory "/" as opposed to `index.html`). I'll take care of all of the changes you've requested above and ping you when it's ready. Thanks!

Excellent, thank you for the update @davidmurdoch ! @cuonglt-ync  @davidnguyen88 Please pause on working on this ticket if/when @davidmurdoch finishes his branch. @davidmurdoch we are happy to re-scan the site once live and further iterate 🥇 

Thank you everyone!

**Comment by jlazoff** [comment]
Excellent, thank you @davidmurdoch !

Tagging Sunny to keep him in the loop. 

Truffle is one step closer to optimization!

**Pull request merged**